app/scripts/migrations/057.test.js

```diff
@@ -0,0 +1,193 @@
+import { strict as assert } from 'assert';
+import migration57 from './057';
+
+describe('migration #57', function () {
+  it('should update the version metadata', async function () {
+    const oldStorage = {
+      meta: {
+        version: 55,
```

Comment:
Nit: we usually use the previous version in this test

```suggestion
        version: 56,
```

Reply:
Oops! thanks